backend/src/schema/resolvers/statistics.js

```diff
@@ -33,10 +33,10 @@ export default {
          * Note: invites count is calculated this way because invitation codes are not in use yet
          */
         response.countInvites = response.countEmails - response.countUsers
+        return response
       } finally {
         session.close()
       }
-      return response
     },
   },
 }
```

Comment:
same as :point_up_2: 
would give greater confidence if we had a test suite for this resolver